### PR TITLE
#164694213 Integrate Coveralls code coverage service

### PR DESCRIPTION
**Pull request by ghost**
#### What does this PR do?
- Integrate Coveralls code coverage service
- Add .coveragerc file
##### Why is this important?*
- This is to ensure we are properly informed just how well our code base is tested which in turn provides quality assurance for the product
#### How should this be manually tested?
- clone the repo to locally.
#### What are the relevant pivotal tracker stories?
#164694213

### CODE REVIEW

**Comment by kimbugp** [comment]
@ManuelDominic ,could you also add the coveralls badge to the readme file

**Comment by kimbugp** [comment]
@ManuelDominic , please rebase with develop

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 Authors Haven - A Social platform for the creative at heart.
 =======
+[![Build Status](https://travis-ci.org/mirest/ah-haven-space-sprinters.svg?branch=develop)](https://travis-ci.org/mirest/ah-haven-space-sprinters) [![Coverage Status](https://coveralls.io/repos/github/mirest/ah-haven-space-sprinters/badge.svg?branch=develop)](https://coveralls.io/github/mirest/ah-haven-space-sprinters?branch=develop) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/396fc4ef974d44a5a6045406ebbe4130)](https://www.codacy.com/app/ManuelDominic/ah-haven-space-sprinters?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=mirest/ah-haven-space-sprinters&amp;utm_campaign=Badge_Grade)
```
Comment (kimbugp) [inline README.md:1]: @ManuelDominic , only add bagdes for codeclimate and coveralls here

File: authors/apps/authentication/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.1.7 on 2019-03-19 12:02
```
Comment (kimbugp) [inline authors/apps/authentication/migrations/0001_initial.py]: @ManuelDominic , this miggration file is already exisitng in develop
Please remove it

File: authors/apps/authentication/urls.py [hunk 3]
```diff
@@ -1,11 +1,11 @@
-from django.conf.urls import url
+from django.urls import path
```
Comment (kimbugp) [inline authors/apps/authentication/urls.py]: @ManuelDominic , all these fixes have been done in the PR #3 .

File: authors/apps/authentication/urls.py [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 
 from .views import (
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
 
 urlpatterns = [
-    url(r'^user/?$', UserRetrieveUpdateAPIView.as_view()),
-    url(r'^users/?$', RegistrationAPIView.as_view()),
```
Comment (kimbugp) [inline authors/apps/authentication/urls.py]: @ManuelDominic , all these fixes have been done in the PR #3 .

File: authors/urls.py [hunk 5]
```diff
@@ -13,12 +13,11 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import include, url
+from django.urls import include, path
 from django.contrib import admin
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
-
```
Comment (kimbugp) [inline authors/urls.py]: @ManuelDominic , all these fixes have been done in the PR #3 .

File: requirements.txt [hunk 6]
```diff
@@ -1,6 +1,9 @@
-Django==1.10
-django-cors-middleware==1.2.0
```
Comment (kimbugp) [inline requirements.txt]: @ManuelDominic , all these fixes have been done in the PR #3 .

**Review by kimbugp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:1]: @kimbugp, i only add the two while at the same time just shifted @Rhytah's badges from up next to the other badges.

**Comment by ghost** [comment]
Reconfigured changes locally to the repo

**Pull request closed without merging**